pybossa/core.py

```diff
@@ -283,6 +282,18 @@ def setup_social_networks(app):
         print inst
```

Comment:
yep, keep it and add the new solution so you can check in the log files what happened too.